src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
+
+import { PoolService } from '../../../shared/api/pool.service';
+import { CdTableColumn } from '../../../shared/models/cd-table-column';
+import { CdTableSelection } from '../../../shared/models/cd-table-selection';
+
+@Component({
+  selector: 'cd-pool-list',
+  templateUrl: './pool-list.component.html',
+  styleUrls: ['./pool-list.component.scss']
+})
+export class PoolListComponent implements OnInit {
+  pools = [];
+  columns: CdTableColumn[];
+  selection = new CdTableSelection();
+
+  constructor(
+    private poolService: PoolService,
+  ) {
+    this.columns = [
+      {prop: 'pool_name', name: 'Name', flexGrow: 2},
+      {prop: 'application_metadata_array', name: 'Applications'},
+      {prop: 'pg_placement_num', name: 'Placement Groups', flexGrow: 1},
+      {prop: 'size', name: 'Replica Size', flexGrow: 1},
```

Comment:
Isn't it a best practice to right align numeric cells (better readability)?

This:
```
  10
1000
 100
```

Instead of this:
```
10
1000
100
```